packages/common/src/components/Footer/Link.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const Link = ({ name, url, newTab }) => (
```

Comment:
Make this a `FunctionComponent` and create named export instead of `default export`.

```ts
type Props = {
  name: string;
  newTab: boolean;
  url: string;
};
const Link: FunctionComponent<Prop> = ({ name, url, newTab }) => (
  // ...
);
```